File: Cap9VB/StringExtensions.vb
Public Module StringExtensions
  <System.Runtime.CompilerServices.Extension()> _
  Public Function IsEmptyAfterTrim(ByVal str As String) As Boolean
    Dim aux As String = str
    If aux Is Nothing Then
      Return True
    End If
    Return String.IsNullOrEmpty(str.Trim())
  End Function
End Module

File: Cap9CSharp/StringExtensions.cs
namespace Cap9CSharp
{
    public static class StringExtensions
    {
        public static bool IsEmptyAfterTrim( this string str )
        {
            string aux = str;
            if( aux == null )
            {
                return true;
            }
            return string.IsNullOrEmpty( str.Trim() );
        }
    }
}